deploy/vagrant/Vagrantfile

```diff
@@ -56,6 +60,8 @@ Vagrant.configure('2') do |config|
       lv.cpus = 1
       lv.boot 'network'
       lv.mgmt_attach = false
+      lv.storage :file, :size => '40G'
```

Comment:
Without this, you cannot actually run a workflow that does something like install an OS to disk.